decide-app/src/main/java/com/decide/app/LaunchInterceptorConditions.java

```diff
@@ -81,7 +81,31 @@ public boolean getLaunchInterceptorCondition7() {
     }
 
     public boolean getLaunchInterceptorCondition8() {
-        return true;
+        if(NUMPOINTS < 5) return false;
+        // Check parameters
+        if(PARAMETERS.RADIUS1 < 0) throw new IllegalArgumentException();
+        if(PARAMETERS.A_PTS < 1 || PARAMETERS.B_PTS < 1 || PARAMETERS.A_PTS + PARAMETERS.B_PTS > NUMPOINTS - 3) throw new IllegalArgumentException();
+
+        double d1;
+        double d2;
+        double d3;
+        Point p1;
+        Point p2;
+        Point p3;
+        int interveningPointsA = PARAMETERS.A_PTS + 1;
+        int interveningPointsB = interveningPointsA + PARAMETERS.B_PTS + 1;
+
+        for(int i = 0; i < NUMPOINTS-interveningPointsB; i++){
+            p1 = POINTS[i];
+            p2 = POINTS[i+interveningPointsA];
+            p3 = POINTS[i+interveningPointsB];
+            d1 = distance(p1, p2);
+            d2 = distance(p1, p3);
+            d3 = distance(p2, p3);
+
+            if(d1 > PARAMETERS.RADIUS1*2 || d2 > PARAMETERS.RADIUS1*2 || d3 > PARAMETERS.RADIUS1*2) return true;

```

Comment:
Same as #55, this is insufficient (as in it can return false when it should be true).

Reply:
Will be fixed after #64 is approved.